reposcan/test/test_batchlist.py

```diff
@@ -0,0 +1,54 @@
+"""
+Tests for batch list module.
+"""
+import pytest
+from common.batch_list import BatchList
+
+
+class TestBatchList:
+    """ Batch list tests group. """
+    batch_list = None
+    dummy_items = []
```

Comment:
It seems you can simply set `dummy_items = [{"item": 1}, {"item": 2}, {"item": 3}]` and then remove `setup_dummy_items`.

Reply:
Also it's convention to name constant vars with upper case.